tests/tests_test_workflow/test_integ_workflow/integ_test/test_integ_test_suite_opensearch.py

```diff
@@ -35,9 +35,10 @@ def setUp(self) -> None:
         self.work_dir = Path("test_dir")
 
     @patch("os.path.exists", return_value=True)
+    @patch("test_workflow.integ_test.integ_test_suite_opensearch.IntegTestSuiteOpenSearch.multi_execute_integtest_sh")
```

Comment:
The reason this and below tests updated is the behavior of python mock tests.
We were mocking `IntegTestSuiteOpenSearch.multi_execute_integtest_sh` here so it will apply to all tests intended to run `IntegTestSuiteOpenSearch.multi_execute_integtest_sh` within this class. Patching this to enforce this mock only applied to this method.